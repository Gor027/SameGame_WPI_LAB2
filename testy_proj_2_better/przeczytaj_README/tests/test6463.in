[[[ cc  [[ [   [ c c   c[[cc[  [[
[[c [c ccc[c    [c
[ c[
 ccc [c [[c[c[c[cc c[
[c [cccc c [ c[[[c cccc[c[ cc   [c[c  [c   c[ 
[c c [[  [c[ [cc[ c  [ [ c ccc[ 
 [ [cc  [cc[ [c cc[[ c[c[c  c cc [[c  [ [c [ cc cc[c c     cc[ cc  c  [ 
  [[c   [[
[ c[c[c [[[c [    c[    [ c ccc   [c[c [c[cc[[   ccc[cc[[ [c cc c[[cc[[ [cc[ c 
cc[c c c[ [[c[ [c[[  c  [ [ [[cc[c [[c [[ccc [c[ c[ [c cc 
c   [[c c[ ccc  [c[cc cc [[cc c  c   [ [[cccc[[ 
c [[cc c [ c  cc[c[c[ c[c cc[c[[cc[   [c
[[ cc[ 
[[ [c ccc c [  [  [ cc c  [c  [ c [[ cc[c[[[c
[[cc[[  c[cc cc [ cc[[[cc[c[c [ [c[[ c[[  cc[ [ccc c[c  c[ [[  cc c c  c[
 c[ cc[c c   c[ c[c[ c [c[[[[cc cc    c [  [[  c[   cc[ [[[cccc [[  
[
[[c[[c cc [c[cc[ccc   [ [c  [ c[c c [[[[ cc[  [c[ c[  [
