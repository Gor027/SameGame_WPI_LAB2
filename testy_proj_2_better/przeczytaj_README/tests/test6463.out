     c
    cc
   [[[   [ [
   ccc  [c c    [           [
   [c[  c[ [ [  ccc         c [
[ [ccc  [c[[ c  [c[c   c[   c c            c
[[c[c[  [[c[ [c[[ccc   [c[ccc c [      c   [
[[c[[ccc[cc[ [[[cc[c c [[[ccc c [c[   [[   c
[cc[c[cc[[cc [c[cc[[[[cc[c[c[ c [[[c c[[[ccc[
[c[ccc[c[[[[c[cc[[[[cc[c[c[ccc[[cc[c c[[c[cc[ c      c
[cccc[ccc[[cccc[[c[cccc[c[cc[c[ccc[c [cc[cc[ccc[cc[c[c  [  c    c
c[c[[[ccc[cccccc[ccc[cccc[[cc[[c[c[[[c[[[c[cc[[ccccc[cc c  cc[  c
cc[cccccc[cc[c[[c[ccc[[cc[c[[cc[cc[[[cc[c[c[[[[c[c[c[c[[c[[[[cccc[[cc [ [
ccc[[c[ccc[c[cc[ccc[[c[[[c[[[cc[ccc[[[[[[cc[[[[c[cc[cc[c[[[ccccc[[ccc[[c[cc[c
